packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -120,43 +179,146 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      className={cx(
-        'iui-expandable-block',
-        {
-          'iui-with-caption': !!caption,
-          'iui-expanded': expanded,
-          'iui-small': size === 'small',
-          'iui-borderless': styleType === 'borderless',
-        },
-        className,
-      )}
-      style={style}
-      ref={ref}
+      className={cx('iui-header', className)}
+      role='button'
+      aria-expanded={isExpanded ?? !disabled}
+      aria-disabled={disabled}
+      tabIndex={0}
+      onClick={handleToggle}
+      onKeyDown={onKeyDown}
+      ref={forwardedRef}
       {...rest}
     >
+      <SvgChevronRight className='iui-icon' aria-hidden />
```

Comment:
if this icon is not customizable, then maybe it should be moved into the css. 🤔

Reply:
Yea I was also thinking about making it customizable. Updated. But then it also made the code too extra since it is part of header (aka if header has children then all the label/title code must also be added). So I decided to make `expandIcon` a convenience prop of `Header`, and if it is passed in then we would just provide the default icon. Otherwise user can override with the `ExpandIcon` subcomponent to use customize the icon they want.